<launch>

    <node name="image_publisher" pkg="image_transport_tutorial" type="image_publisher"
          args="$(find ml_datasets)/val_datasets/test_images/" output="screen">
          <param name="pubTopic" value="/recogImage" />
          <param name="pubRate"  value="0.3" />
    </node>

    <node name="image_view" pkg="image_view" type="image_view"  args="image:=/machine_learning/detection_test" />

</launch>
